aws/resource_aws_codebuild_project.go

```diff
@@ -728,10 +728,11 @@ func validateAwsCodeBuildEnvironmentType(v interface{}, k string) (ws []string,
 func validateAwsCodeBuildSourceType(v interface{}, k string) (ws []string, errors []error) {
 	value := v.(string)
 	types := map[string]bool{
-		"CODECOMMIT":   true,
-		"CODEPIPELINE": true,
-		"GITHUB":       true,
-		"S3":           true,
+		codebuild.SourceTypeCodecommit:   true,
+		codebuild.SourceTypeCodepipeline: true,
+		codebuild.SourceTypeGithub:       true,
+		codebuild.SourceTypeS3:           true,
+		codebuild.SourceTypeBitbucket:    true,
 	}
 
 	if !types[value] {
```

Comment:
I think we should also update the error message below - perhaps we could make it future-proof and print allowed types by converting the map above to a slice and printing it via `strings.Join(s, ", ")`?